### PR TITLE
New docs site content

### PR DESCRIPTION
**Pull request by arnoldsandoval-okta**
Adds new docs site content.

Please note that *a lot of this content is a work in progress* and features a variety of placeholder content for text and images.

### CODE REVIEW

File: packages/docs/components/fpo.svg [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<svg>
+  <rect width="100%" height="100%" style="fill: #fff; stroke:#a0a0a0;"></rect>
```
Comment (edburyenegren-okta) [inline packages/docs/components/fpo.svg:2]: We should prolly move these colors to a class on the `svg`

**Review by edburyenegren-okta — APPROVED** [review]
Pulled to my local with some finagling - looks good!

One suggestion inline. One other nit: we should run all of our production SVGs through SVGO. 

Since this is all WIP content, I'm 💯 on leaving it here and optimizing when we pull in the rewrites and final assets. Your call!

File: packages/docs/components/fpo.svg (hunk 1 above)
Comment (arnoldsandoval-okta) [reply packages/docs/components/fpo.svg:2]: this should never be visible, its just a placeholder deal - will be deleted before we release.

File: packages/docs/help.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+</Description>
+
+<Description>
+
+## Github
+
+Code for SCSS, tokens, icons, and the documentation lives in our Odyssey repo. We also do all of our project planning there too.
+
+<button class="ods-button">Open GitHub</button>
```
Comment (jeffbelser-okta) [inline packages/docs/help.md:26]: are these buttons suppose to be wired up?

**Review by jeffbelser-okta — APPROVED** [review]

**Comment by jeffbelser-okta** [comment]
Also - the new docs site looks very pleasant! 🌀🌀🌀

File: packages/docs/components/tag.md [hunk 3]
```diff
@@ -1,27 +1,167 @@
...
+  - icon: github
+    label: View source
+    href: https://github.com/okta/odyssey/blob/master/packages/odyssey/src/scss/components/_tag.scss
+  - icon: figma
+    label: View designs
+    href: https://www.figma.com/file/AQcK0mLE4JHBm2Vci8ulAF/Tag?node-id=25%3A2
+---
 
-# Tag
+::: slot overview
```
[the remark this answers is not in the document]
Comment (jeffbelser-okta) [reply packages/docs/components/tag.md:21]: Nvm I can see now this is scripting code for vuepress itself

**Pull request merged**